src/DIRAC/Interfaces/API/DiracAdmin.py

```diff
@@ -157,34 +155,30 @@ def getSiteSection(self, site, printOutput=False):
 
     #############################################################################
     def allowSite(self, site, comment, printOutput=False):
-        """Adds the site to the site mask.
+        """Adds the site to the site mask. The site must be a valid DIRAC site name
 
         Example usage:
 
-          >>> gLogger.notice(diracAdmin.allowSite())
+          >>> gLogger.notice(diracAdmin.allowSite('LCG.CERN.ch'))
           {'OK': True, 'Value': }
 
+        :param str site: DIRAC site name
+        :param str comment: comment for the site status update
         :return: S_OK,S_ERROR
 
         """
-        result = self._checkSiteIsValid(site)
-        if not result["OK"]:
+        if not (result := self._checkSiteIsValid(site))["OK"]:
             return result
 
-        result = self.getSiteMask(status="Active")
-        if not result["OK"]:
+        if not (result := self.getSiteMask(status="Active"))["OK"]:
             return result
         siteMask = result["Value"]
```

Comment:
`result` is the previous `result` here if you use this walrus operator no ?

Reply:
Mmm no...

```py
if not (result := self._checkSiteIsValid(site))["OK"]:
    return result

if not (result := self.getSiteMask(status="Active"))["OK"]:
    return result
```

and

```py
result = self._checkSiteIsValid(site)
if not result["OK"]: 
    return result

result = self.getSiteMask(status="Active")
if not result["OK"]:
    return result
```

seem logically equivalent to me (and also according to chatGPT)